### PR TITLE
adds search capabilities

### LINKED ISSUE #432: add search capabilities
Ensure search bar in header actually leads to search functionality.

### PR DESCRIPTION
**Pull request by MarquiseRosier**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #432 

Test URLs:
- Before: 
  - https://main--shredit--stericycle.aem.page/en-us/search
- After: 
  - https://adding-search--shredit--stericycle.aem.live/en-us/search

### CODE REVIEW

**Comment by kronnox** [comment]
Is there any reason you don't use our version of `ffetch` that already has the local storage caching and stuff in place?

File: blocks/search/search.css [hunk 1]
```diff
@@ -0,0 +1,276 @@
+/* search box */
+.search .search-box {
```
Comment (bstopp) [inline blocks/search/search.css:2]: all the `.search` references should be `.search.block` or `.block.search`

**Review by bstopp — CHANGES_REQUESTED** [review]
Need a better way to indicate no results. If there aren't any results, there shouldn't be any pagination

Example: https://adding-search--shredit--stericycle.aem.live/en-us/search?searchQuery=principal

Are we ditching the ability to jump to the last page, or show how many pages there are in the results?

File: blocks/search/search.css (hunk 1 above)
Comment (MarquiseRosier) [reply blocks/search/search.css]: @kronnox done!

**Review by bstopp — CHANGES_REQUESTED** [review]
If not running the search (i.e. term is too short "te"), then something should be displayed, even if it's "not enough input"

Live site actually just shows "No results" message.

**Comment by MarquiseRosier** [comment]
@bstopp that 

> If not running the search (i.e. term is too short "te"), then something should be displayed, even if it's "not enough input"
> 
> Live site actually just shows "No results" message.

@bstopp that should be resolved now!

**Review by bstopp — APPROVED** [review]

**Pull request merged**